code/modules/mob/living/simple_animal/hostile/hostile.dm

```diff
@@ -177,7 +183,7 @@ mob/living/simple_animal/hostile/hitby(atom/movable/AM as mob|obj,var/speed = TH
 		var/obj/machinery/porta_turret/T = target_mob
 		src.do_attack_animation(T)
 		T.take_damage(max(melee_damage_lower, melee_damage_upper) / 2)
-		visible_message("<span class='danger'>\The [src] [attacktext] \the [T]!</span>")
+		visible_message(SPAN_DANGER("[capitalize_first_letters(src.name)] [attacktext] \the [T]!"))
```

Comment:
I don't know about this:
`The Space Carp attacks Unga Bunga!`
Eeeeh. Probably fine really.